src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -36,7 +36,7 @@
                                 <FluentTreeItem Class="metrics-tree-item" Text="@meterGroup.Key.MeterName" Data="@meterGroup.Key" title="@meterGroup.Key.MeterName" InitiallyExpanded="true" InitiallySelected="@(_selectedInstrument == null && meterGroup.Key.MeterName == _selectedMeter?.MeterName)">
```

Comment:
```suggestion
                                <FluentTreeItem @key="@meterGroup.Key" Class="metrics-tree-item" Text="@meterGroup.Key.MeterName" Data="@meterGroup.Key" title="@meterGroup.Key.MeterName" InitiallyExpanded="true" InitiallySelected="@(_selectedInstrument == null && meterGroup.Key.MeterName == _selectedMeter?.MeterName)">
```
I tested and it is required to stop the first level tree items having the same selection issue.

Unfortunately it introduced a rendering quirk where second level items jump slightly when loaded:

![metrics-tree-jump](https://github.com/dotnet/aspire/assets/303201/370b91ba-3015-4b72-b084-867df0e67fb1)

Is that from items being shown and then web component upgrade?

Reply:
It shouldn't be web component upgrade - that issue only really happens when the JS isn't loaded when the elements get rendered. 

I _think_ what is happening is when tree items are added as children to tree items, they're [marked as nested](https://github.com/microsoft/fast/blob/2ea80697bc3a5193e6123fb08ac3be2a76571aeb/packages/web-components/fast-foundation/src/tree-item/tree-item.ts#L119) and then the templating for the tree item runs again and that adds `nested` to the class. I'd think that would be fast enough we wouldn't notice the rendering, but maybe not. 

Another thought is that this could be another issue similar to https://github.com/microsoft/fluentui-blazor/pull/1082 - a similar situation is happening here where FluentTreeItem is [setting the class attribute](https://github.com/microsoft/fluentui-blazor/blob/255d8db80d37c4bad0319bff2c8f895715add78e/src/Core/Components/TreeView/FluentTreeItem.razor#L4) but the underlying tree item is setting it for itself as well (see above). Hrm...

Reply:
Alright, I think I found something that works. If I add nested to the ones we know are nested, then it no longer jumps:

![MetricsTreeJumpingNoMore](https://github.com/dotnet/aspire/assets/9613109/8df4d02c-7082-4e22-ab06-5039d7fd2abb)